tools/model_converters/efficientnetv2-timm_to_mmcls.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) OpenMMLab. All rights reserved.
```

Comment:
just name the script as `efficientnetv2_to_mmcls`.

add a comment :
```
""" 
convert the weights of efficientnetv2 in timm(HTTP://,.....................) to mmcls format.
"""
```

Reply:
Done